scatter_test.py

```diff
@@ -0,0 +1,12 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+
+midwest = pd.read_csv("http://goo.gl/G1K41K")
+midwest = midwest[midwest['poptotal'] < 50000]
+graph = midwest.plot(kind='scatter',
+                     x='area',
+                     y='poptotal',
+                     s='popdensity',
+                     s_grow=0.2,
+                     title='Popuation vs area and density')
+plt.show()
```

Comment:
We only create new test file if it is absolutely necessary.  Surely this test has a home under an existing module in the `pandas/tests` directory.

Reply:
Also, I'm imagine that this test will probably have to be rewritten slightly (correct me if I'm wrong @TomAugspurger ).

Reply:
Yep, this can go in `plotting/test_frame.py`. Then you have all the plotting infrastructure in place.

If possible, avoid using a new dataset. We have some in `pandas/tests/io/data` and some in `doc/data`

Reply:
I rewrote the test to check that the sizes of the bubbles in the plot are related to the data in the size column as expected, and placed it in a new function in plotting/test_frame.py.
Let me know if it's OK now.